src/const.js

```diff
@@ -1,3 +1,9 @@
+import {
```

Comment:
I had this discussion with @muffinresearch. It's possible but we'd be moving messages out of messages which seemed like it would also seemed like a poor idea.